index.js

```diff
@@ -12,6 +12,18 @@ app.get('/api/getList', (req,res) => {
     res.json(list);
     console.log('Sent list of items');
 });
+app.get('/api/getAllBookings', (req,res) => {
```

Comment:
maybe you should look into how to make multiple routers in express, and a prefix for these routes. [check this link out for the documentation](https://expressjs.com/en/guide/routing.html#express-router)